build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+export VERSION=1.3.2
+export BUILD_NUMBER=acquia2
+export TARGET=Net_DNS2-${VERSION}
+export ARTIFACT=${TARGET}-${BUILD_NUMBER}.tgz
+
+set -ex
+
+# Set up the proper directory structure
+rm -fr ./build
+mkdir -p ./build/Net_DNS2-$VERSION
+cd build
+
+# Move source files into place.
+cp -rv ../Net ./${TARGET}/
+cp -rv ../tests ./${TARGET}/
```

Comment:
Why we need tests on the build artifact?

Reply:
Great question.  This is a PEAR library and PEAR libraries include the tests.  Continuing to include them minimizes the differences between our new artifact and what is in production today.